Solutions/Marain.TenantManagement.Specs/Steps/TenantSteps.cs

```diff
@@ -40,18 +40,25 @@ public Task WhenIUseTheTenantManagementServiceToCreateANewClientTenantCalled(str
                 });
         }
 
+        [Given(@"I have an existing client tenant with a well known Guid '(.*)' called '(.*)'")]
```

Comment:
We normally try and remove the `@` if they aren't needed. Roslynator should be able to fix all of these for you in one go.

Reply:
Removed